tools/spectral/.spectral.yaml

```diff
@@ -16,18 +16,6 @@ aliases:
     - "$.components.schemas[*]..properties[?(@ && @.type)]"
 
 rules:
-  xgen-schema-properties-camel-case:
-    description:
-        All properties in the schema must be camelCase. http://go/ipa/112
-    severity: error
-    recommended: true
-    message: "`{{property}}` MUST follow camelCase"
-    given: "$.components.schemas..properties[*]~"
-    then:
-      function: pattern # casing?
-      functionOptions:
-        match: "/^[a-z$_]{1}[A-Z09$_]*/" # type: camel ?
```

Comment:
[To ensure] Is that regexp exactly the same that the one we have or it is least restrictive?